frontend/src/model/beta/reviews/ReviewBanner.tsx

```diff
@@ -85,6 +87,7 @@ export default function ReviewBanner({ release, accessRequest, square = false }:
       mutateReleases()
       mutateAccessRequests()
       setIsReviewOpen(false)
+      setLoading(false)
```

Comment:
This should be moved outside of this `if` statement. We want to `setLoading(false)` regardless of whether or not we get a success/error response. We're just using `loading` to track the time spent waiting for the API to respond.